docs/remoteExams.md

```diff
@@ -51,7 +51,7 @@ This configuration is good when:
 
 Some notes about this configuration:
 
-* The exam window (70 minutes, `startDate` to `endDate`) has been set to be 10 minutes longer than the exam time limit (60 minutes). If a student starts the exam late, then the countdown timer on their exam will show the full exam time limit (60 minutes). However, they will not be able to access the exam past the `endDate` time under any circumstances. See [PL issue #2217](https://github.com/PrairieLearn/PrairieLearn/issues/2217) for details.
+* The exam window (70 minutes, `startDate` to `endDate`) has been set to be 10 minutes longer than the exam time limit (60 minutes). However, they will not be able to access the exam past the `endDate` time under any circumstances. If a student starts this exam more than 10 minutes late, then the countdown timer on their exam will reflect the time remaining until `endDate` (previously, the timer would incorrectly show the full exam time limit regardless of `endDate`).
```

Comment:
```suggestion
* The exam window (70 minutes, `startDate` to `endDate`) has been set to be 10 minutes longer than the exam time limit (60 minutes). However, students will not be able to access the exam past the `endDate` time under any circumstances. If a student starts this exam more than 10 minutes late, then the countdown timer on their exam will reflect the time remaining until `endDate`.
```

I think it's best to just describe how things currently work. Talking about how things used to be is helpful in the short run, but it quickly gets confusing over time.